opencsp/app/sofast/lib/ProcessSofastFringe.py

```diff
@@ -649,6 +649,7 @@ def get_optic(
             Optic object

```

Comment:
Not directly related to these changes. Suggestion for doc string:
```
Returns
-------
FacetEnsemble if ProcessSofastFringe.optic_type = 'multi', otherwise Facet